Source/Core/Core/PowerPC/Jit64Common/Jit64AsmCommon.cpp

```diff
@@ -75,24 +72,53 @@ void CommonAsmRoutines::GenFrsqrte()
   MOVQ_xmm(XMM0, R(RSCRATCH2));
   RET();
 
-  // Exception flags for zero input.
-  SetJumpTarget(zero);
+  SetJumpTarget(complex);
+  AND(32, R(RSCRATCH_EXTRA), Imm32(0x7FF));
+  CMP(32, R(RSCRATCH_EXTRA), Imm32(0x7FF));
+  FixupBranch nan_or_inf = J_CC(CC_E);
+
+  MOV(64, R(RSCRATCH2), R(RSCRATCH));
+  SHL(64, R(RSCRATCH2), Imm8(1));
+  FixupBranch nonzero = J_CC(CC_NZ);
+
+  // +0.0 or -0.0
   TEST(32, PPCSTATE(fpscr), Imm32(FPSCR_ZX));
-  FixupBranch skip_set_fx1 = J_CC(CC_NZ);
+  FixupBranch skip_set_fx3 = J_CC(CC_NZ);
```

Comment:
Just wondering, i don't see `skip_set_fx1` anywhere...any reason this is number three?